Sources/StreamChat/Database/DTOs/ChannelConfigDTO.swift

```diff
@@ -41,8 +41,8 @@ final class ChannelConfigDTO: NSManagedObject {
             commands: Array(Set(
                 commands.compactMap { try? ($0 as? CommandDTO)?.asModel() }
             )),
-            createdAt: createdAt,
-            updatedAt: updatedAt
+            createdAt: createdAt.bridgeDate,
```

Comment:
I'm fine with `bridgeDate` but IMO `toDate()` or `asDate()` would be more consistent with the rest of the SDK + it would be easier to understand and use 🤔